package.json

```diff
@@ -16,13 +16,15 @@
     "bcrypt": "5.1.1",
     "dotenv": "16.4.05",
     "express": "4.18.2",
+    "joi": "^17.13.3",
```

Comment:
joi is a pretty cool validation schemas but in this project we will use zod, please check the recent pr merged https://github.com/Ditmar/api-openscience/pull/18/files